app/javascript/dashboard/components/ChatList.vue

```diff
@@ -965,9 +965,6 @@ export default {
     flex-basis: clamp(20rem, 4vw + 21.25rem, 27.5rem);
   }
 }
-</style>
-
-<style scoped lang="scss">
```

Comment:
@pranavrajs Inside this tag `<style scoped lang="scss">`  there is some nested CSS. Will it work when we remove it?

Reply:
I'm not sure. Let me test it out. It was giving an error that the layer is not defined.

Reply:
I tried testing it's breaking some v-deep styles. We can move `flex-basis-clamp` layer style to inline from the styles. I have moved it [here](https://github.com/chatwoot/chatwoot/pull/7617/commits/dcc7bc1c3293c004b4721f7e5f107ff93942042b). Hope it will fix.